apps/backend/src/services/accounts.js

```diff
@@ -345,155 +387,64 @@ async function hydrateDemoData({ userId }) {
       }
     })
 
-  // create demo session without results
-  const demoSession = new SessionModel({
-    settings: {
-      isParticipantAuthenticationEnabled: false,
-      isConfusionBarometerActive: false,
-      isEvaluationPublic: false,
-      isFeedbackChannelActive: false,
-      isFeedbackChannelPublic: true,
-      authenticationMode: 'NONE',
-      storageMode: 'SECRET',
+  const feedbacks = [
```

Comment:
we could also add feedbacks, and they would show up immediately on the running session cockpit (so there is some data there). confusion values would work as well, but would be filtered out of the chart